packages/oidc-dpop-client-browser/src/cleanup/cleanup.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { OidcClient, WebStorageStateStore } from "oidc-client";
+
+/**
+ * Removes OIDC-specific query parameters from a given URL (state, code...)
+ * @param redirectUrl The URL to clean up.
+ * @returns A copy of the URL, without OIDC-specific query params.
+ */
+export function cleanupRedirectUrl(redirectUrl: string): string {
```

Comment:
This function name could be more specific (e.g. it's not really redirect-URL-specific at all (i.e. it just takes in any string (and I guess could throw if that string is an invalid URL!?)), and it specifically and explicitly removes very specific query params (as opposed to all possible OIDC-specific query params). I know this whole PR is a horrific hack, but I'd rename this function just in case this code survives more than a few days!
```suggestion
export function removeQueryParamCodeAndState(url: string): string {
```

Reply:
I kept the name generic, because I expect the same issue will apply in the implicit flow with `id_token` and `access_token`, but the function could be named `removeOidcQueryParam`

Reply:
Fixed in ea0e3a72b23994927eb8b8869daf85e5352fbabe